hclsyntax/expression_vars_gen.go

```diff
@@ -87,7 +87,6 @@ func main() {
 
 }
 
-const outputPreamble = `// Copyright (c) HashiCorp, Inc.
```

Comment:
This should not be deleted; instead it should be updated with the new header. Since the copyright header is automatically populated, a better PR would be to fix this file manually and re-run the generated headers by running `make copyrightfix`